packages/twenty-front/src/modules/settings/accounts/components/SettingsAccountsMessageChannelDetails.tsx

```diff
@@ -111,6 +119,8 @@ export const SettingsAccountsMessageChannelDetails = ({
             }}
           />
           <SettingsOptionCardContent
+            variant="toggle"
```

Comment:
I'm not sure about this variant="toggle" | "select" | "number" modifying the props. We don't do that in the project, we prefer having specific components
Why not having 3 components: SettionsOptionCardContentToggle, SettingsOptionCardContentSelect, ... and put common props in a common type?